src/screens/ExchangeModal.tsx

```diff
@@ -352,7 +406,7 @@ export default function ExchangeModal({
     )('transitionEnd', ({ data: { closing } }) => {
       if (!closing && isDismissing.current) {
         isDismissing.current = false;
-        lastFocusedInputHandle?.current?.focus();
+        ((lastFocusedInputHandle as unknown) as MutableRefObject<TextInput>)?.current?.focus();
```

Comment:
@estebanmino @jxom using this type assertion feels a bit gross in the long term but as alot of the underlying code isnt typed, im having to type check it here.

Reply:
i believe this one is a `React.MutableRefObject` since is coming from `useMagicAutofocus` hook

Reply:
ah yes you put it there, i'm blind